src/pmdas/bpftrace/bpftrace/process_manager.py

```diff
@@ -81,14 +81,16 @@ async def stop_bpftrace_process(self, script: Script, script_tasks: ScriptTasks)
         process.send_signal(signal.SIGINT)
 
         # wait max. 5s for graceful termination of the bpftrace process
-        _done, pending = await asyncio.wait({process.wait()}, timeout=5)
```

Comment:
`process.wait()` returns a coroutine: https://docs.python.org/3/library/asyncio-subprocess.html#asyncio.subprocess.Process.wait

The error reads "Passing coroutines is forbidden, use tasks explicitly." (I assume this is a new rule in some recent Python version), can you try wrapping the coroutine in a `Task`?

```
_done, pending = await asyncio.wait({asyncio.create_task(process.wait())}, timeout=5)
```

Reply:
Good call - something like the last update?